buildkite/src/Jobs/Lint/Fast/Pipeline.dhall

```diff
@@ -29,6 +34,15 @@ Pipeline.build
         key = "lint",
         target = Size.Small,
         docker = Some Docker::{ image = (../../../Constants/ContainerImages.dhall).toolchainBase }
+      },
+    Command.build
+      Command.Config::{
+        commands = compat_diff_commands,
+        label = "Optional lint steps; versions and binable compatability changes",
+        key = "lint",
```

Comment:
You'll need to change this key -- see https://buildkite.com/o-1-labs-2/coda/builds/300#b4e7eeb8-ff1c-430b-a9bd-e2f62a0dd84d

Since we have more than one "fast lint" command step now, we should probably rename the other one too. Maybe "required" and "optional" ?

Reply:
I wonder if it's time to move the stuff in `lint-opt` to `lint`, that is, make it required.